breaking-front/src/components/ProfileSettingForm/ProfileSettingForm.js

```diff
@@ -105,11 +108,20 @@ export default function ProfileSettingForm({
       } else formData.append('profileImg', profileImgURL);
     }
 
-    formData.append('signUpRequest', JSON.stringify(userData));
+    if (username) formData.append('signUpRequest', JSON.stringify(userData));
+    else formData.append('updateRequest', JSON.stringify(userData));
```

Comment:
앞에 if문 3개를 이런식으로 바꾸는건 가능한가요?
```suggestion
    if(수정페이지라면){
      if (이미지 변경하지 않았으면) {
        const imageFile = await urlToFile(profileImgURL);
        formData.append('profileImg', imageFile);
      } else formData.append('profileImg', profileImgURL);
      // 후처리
      userData.username = username;
      formData.append('signUpRequest', JSON.stringify(userData));
    }
    else{
      formData.append('updateRequest', JSON.stringify(userData));
    }
```